README.md

```diff
@@ -89,11 +89,11 @@ foods = {"pie" => "delicious", "broccoli" => "not delicious",
 
 11 . What is the return value of this method?
 ```ruby
-  characters = ["Daenerys Targaryen", "Jon Snow" ,"Arya Stark", "Tyrion Lannister", "Sansa Stark", "Cersei Lannister", "Margaery Tyrell"]
+  character_names = ["Daenerys Targaryen", "Jon Snow" ,"Arya Stark", "Tyrion Lannister", "Sansa Stark", "Cersei Lannister", "Margaery Tyrell"]
 
-  def downcase_names(characters)
-    characters.each do |names|
-      names.downcase
+  def downcase_all(array_of_strings)
+    array_of_strings.each do |name|
```

Comment:
if we want downcase_all to be a generic method that can operate on any array of strings, can we rename the block variable to something more generic like `string`

Reply:
Good call!